package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "@radix-ui/react-dialog": "^1.1.1",
+    "@tanstack/react-query": "^5.52.1",
```

Comment:
is this maintained

Reply:
the tanstack one is, they picked up the old package and implemented a v4

Reply:
https://tanstack.com/query/latest
v5 actually now